lib/pushmi_pullyu/aip_downloader_files_directories.rb

```diff
@@ -0,0 +1,91 @@
+require 'rdf'
```

Comment:
We are not including this in our gemspec as a dependency if we are using it? 

Everything relating to RDF should be done via RDF/N3 gems interface? `RDF::N3`? Else we should be depending on this ourselves (incase RDF::N3 does something weird in the future) and adding RDF to our gemspec.